modules/web/src/com/haulmont/cuba/web/gui/components/WebSplitPanel.java

```diff
@@ -356,34 +358,44 @@ protected Unit convertLegacyUnit(int unit) {
     @Override
     public void setDockable(boolean dockable) {
         if (orientation == SplitPanel.ORIENTATION_VERTICAL) {
-            throw new IllegalStateException("Docking is not available for the vertically oriented SplitPanel.");
+            ((CubaVerticalSplitPanel) component).setDockable(dockable);
```

Comment:
Since both vertical and horizontal split panels support docking, it would be convenient to create an interface with methods like `setDockable()`, `isDockable()`, etc.  Make `CubaVerticalSplitPanel` and `CubaHorizontalSplitPanel` implement this interface.